packages/wrangler/src/cloudchamber/deploy.ts

```diff
@@ -18,14 +18,17 @@ export async function maybeBuildContainer(
 	imageTag: string,
 	dryRun: boolean,
 	pathToDocker: string
-) {
+): Promise<{ image: string; pushed: boolean }> {
 	try {
 		if (
 			!isDockerfile(
 				containerConfig.image ?? containerConfig.configuration?.image
```

Comment:
The type checker is failing to catch this, but the type of `containerConfig.image ?? containerConfig.configuration?.image` is `string | undefined`, but `isDockerfile` only accepts a `string`. This is causing errors at runtime.